src/subscript/field_statistics/field_statistics.py

```diff
@@ -967,50 +1007,120 @@ def get_specifications(input_dict, ertbox_size, ert_config_path):
             f"Missing keyword:  {key} specifying a list of iteration numbers "
             " for ensembles from ERT ES-MDA"
         )
-    key = "use_zones"
-    zone_names_used = copy.copy(list(zone_code_names.values()))
-    if key in input_dict:
-        zone_names_input = input_dict[key]
-        if zone_names_input is not None and len(zone_names_input) > 0:
-            zone_names_used = zone_names_input
-    check_use_zones(zone_code_names, zone_names_used)
 
-    key = "zone_conformity"
-    if key in input_dict:
-        zone_conformity = input_dict[key]
-    else:
-        raise KeyError(f"Missing keyword:  {key} specifying conformity per zone.")
-    check_zone_conformity(zone_code_names, zone_names_used, zone_conformity)
+    if not ertbox_size:
+        # ertbox size does not exist, read it from this scripts config file instead
+        print("ERTBOX size is not defined, need to get it from the config file")
```

Comment:
I think it's best to user logger instead of printing it to screen. Especially for forward model or workflow job.

Reply:
I can fix this